packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PeoplePickerItemSuggestion.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { classNamesFunction, styled } from '../../../../Utilities';
+import { Persona, PersonaSize } from '../../../../Persona';
+import {
+  IPeoplePickerItemSuggestionProps,
+  IPeoplePickerItemSuggestionStyleProps,
+  IPeoplePickerItemSuggestionStyles
+} from './PeoplePickerItem.types';
+import { getStyles } from './PeoplePickerItemSuggestion.styles';
+
+const getClassNames = classNamesFunction<IPeoplePickerItemSuggestionStyleProps, IPeoplePickerItemSuggestionStyles>();
+
+export const PeoplePickerItemSuggestionBase = (props: IPeoplePickerItemSuggestionProps) => {
+  const { personaProps, suggestionsProps, compact, styles, theme, className } = props;
+
+  const classNames = getClassNames(styles, {
+    theme: theme!,
+    className: (suggestionsProps && suggestionsProps.suggestionsItemClassName) || className
+  });
+
+  return (
+    <div className={classNames.root}>
+      <Persona size={PersonaSize.size24} className={classNames.personaWrapper} showSecondaryText={!compact} {...personaProps} />
+    </div>
+  );
+};
+
+export const PeoplePickerItemSuggestion = styled<
```

Comment:
This is technically a breaking change, you should still probably export a CompactPeoplePickerItemSuggestion that passes through the isCompact prop.

Reply:
Not sure if that will be helpful because of arguments type being different from the previous implementation. Even if I re-export these under the old name these function are still incompatible with old ones.
All these subComponents were designed to be a `defaultRenderer` for `PeoplePicker` items. We never exported them and we never passed them to the `onRender...` callbacks.